lib/validate_config.js

```diff
@@ -0,0 +1,13 @@
+var CLIEngine = require("eslint").CLIEngine
+  , cli = new CLIEngine()
+  , fs = require("fs");
+
+module.exports = function(configPath) {
+  if (configPath) {
```

Comment:
This behavior is a little surprising to me: this would mean if you explicitly told us to use `.eslintrc.foo` as a config file, but that file didn't exist, we would still pass that as valid?